### ISSUE TITLE
inference speed 250 fps

### ISSUE THREAD

**Issue by yangshunDragon** [ZhengPeng7/GCoNet_plus#1]
Hello, thanks for nice project and you had done solid work.

I have some about the inference time measurement. 
I also write script to check the inference time.
while my result is 10 ms per image(100 fps). 
It is only inference time, the time of data loading and image resizing are not included.

I am wondering how to make 250 fps.
My hardware are 2080ti GPU and i7 cpu.
Thanks in advance!

**Comment by ZhengPeng7** [comment]
Thanks for your interest.
Yeah, the speed you obtained is right. Testing on a single 2080Ti costs about 10ms on a pair of images, i.e., batch size = 2 as we denoted in the paper. But we test the speed on a single Tesla V100.
I will set up a colab-notebook as a demo to help with your test, but just too busy these days. I'll @you when it's done.

**Comment by ZhengPeng7** [comment]
There was chaos in the inference speed test of CoSOD models, which made it harder to set up a fair comparison:
1. too many previous projects may have bugs;
2. fixed parameters (input size, batch size, some group size, ...);
3. different GPUs are used for the test.

Hence, we made a repo for the fair fps comparison of CoSOD models: https://github.com/ZhengPeng7/CoSOD_fps_collection, where I made all models inference with:
1. batch size 2;
2. input size 256x256;
3. all tested on a single A100 GPU;
4. the inference is conducted 1000 times to get the average performance for stability;
5. only inference codes are kept for the simplicity of this repo.
